.github/workflows/npm-release.yml

```
@@ -0,0 +1,93 @@
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "22"
+          registry-url: 'https://registry.npmjs.org'
+      - id: node-release
+        name: Bump version and create git tag
+        working-directory: ./${{inputs.app-path}}
+        run: | 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)